website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -4,391 +4,275 @@ linkTitle: Releasing
+### Before starting
 
-Submit a PR with the changes and merge it with master.
+Create and merge a commit that updates the top-level [Makefile] variable `IMAGE_VERSION`
+to the upcoming release tag `v1.3.1`. This variable ensures sample projects have been tagged
+correctly prior to the release commit.
 
-**Important:**
-- Update OLM bindata just before starting the release so that we have the latest OLM version.
-- Verify that the release of OLM version which you specify in Makefile is successful.
+### 1. Create and push a release commit
 
-### 2. Netlify configuration
-
-**Important:** ensure a release branch-to-subdomain mapping exists in the SDK's Netlify configuration _prior to creating a release_,
-ex. `v1.3.x` to `https://v1-3-x.sdk.operatorframework.io`. You can ping SDK [approvers][doc-owners] to ensure a
-[release branch](#release-branches) is created prior to the release and that this mapping is created.
-
-### 3. Create release branch for Netlify
-
-The release branch must be created before the release occurs to appease the Netlify website configuration demons.
-You can do so by running the following before proceeding with the release, assuming the upstream SDK is the `upstream` remote repo:
+Create a new branch from the release branch, which should already exist for the desired minor version,
+to push the release commit to:
 
 ```sh
-$ git checkout master
-$ git pull
-$ git checkout -b v1.3.x
-$ git push -u upstream v1.3.x
+git checkout v1.3.x
+git pull
+git checkout -b release-v1.3.1
 ```
 
-### 4. Create a PR for release version, CHANGELOG.md, and migration guide updates
-
-Once all PR's needed for a release have been merged, branch from `master`:
+Run the pre-release `make` target:
 
 ```sh
-$ git checkout master
-$ git pull
+make prerelease RELEASE_VERSION=v1.3.1
 ```
 
-If making a patch release, check out the corresponding minor version branch:
+The following changes should be present:
 
-```sh
-$ git checkout v1.2.x
-$ git pull
-```
+- `changelog/generated/v1.3.0.md`: commit changes (created by changelog generation).
+- `changelog/fragments/*`: commit deleted fragment files (deleted by changelog generation).
 
-Create a new branch to push release commits:
+Commit these changes and push:
 
 ```sh
-$ git checkout -b release-v1.3.0
+git add --all
+git commit -m "Release v1.3.1"
+git push -u origin release-v1.3.1
 ```
 
-Run the CHANGELOG and migration guide generator:
+### 2. Lock down branch `v1.3.x`
 
-```sh
-$ GEN_CHANGELOG_TAG=v1.3.0 make changelog
-```
+Lock down the `v1.3.x` branch to prevent further commits before the release completes.
+See [this section](#locking-down-branches) for steps to do so.
```

Comment:
I realize this will be duplicating information, but I would put the locking down steps INLINE here. Jumping around during a release is annoying and error prone. When I did the releases I'd go step by step paying close attention to the code blocks.